/**:
  ros__parameters:
    device_port: "/dev/ttyS2"
    baudrate: 115200
    slave_id: 9
    publish_rate: 20.0
